src/python/pants/engine/internals/native_engine.pyi

```diff
@@ -327,7 +327,7 @@ class PySession:
         self,
         *,
         scheduler: PyScheduler,
-        should_render_ui: bool,
+        ui_renderer: str | None,
```

Comment:
To reduce the number of places we have to worry about the enum values (and because I don't expect a third), it would probably be easiest for this to be `ui_use_prodash: bool`.

Reply:
You still OK with the Python enum? I think that should stay, considering the naming.

Reply:
Well, actually we might need an enum, because otherwise this becomes a tri-bool (otherwise we can have 2 params, but thats code smell IMO)

Reply:
Ehhhhhhhhh two params wont' kill anyone